addons/mass_mailing/static/src/fields/html_field/mass_mailing_html_field.js

```diff
@@ -321,6 +324,22 @@ export class MassMailingHtmlField extends HtmlField {
             .catch(() => (this.isDirty = true));
         this.props.record.model.bus.trigger("FIELD_IS_DIRTY", this.isDirty);
     }
+    /**
+     *
```

Comment:
maybe you wanted to add an explanation? I would at least mention at TODO EGGMAIL: move in a `convert_inline` plugin once they are implemented (for my own refactoring)